docs/LicenseTerms/Microsoft .NET Portability Analyzer CLI Third Party Notices.txt

```diff
@@ -15,6 +15,7 @@ implication, estoppel or otherwise.
 2.	Autofac.Configuration  (https://autofac.org/)
```

Comment:
Thanks Jeremy for preemptively adding this.

Reply:
It was the compliance drive.

---
In reply to: [178088762](https://github.com/Microsoft/dotnet-apiport/pull/598#discussion_r178088762) [](ancestors = 178088762)